### PR TITLE
CON-2277 Create new subject and audit for `play-with-containers-py` based on JS version

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

File: subjects/devops/play-with-containers-py/audit/README.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+user:~$
+```
+
+###### Does the student architecture reflect the infrastructure enforced by the subject?
+
+###### Does the infrastructure start correctly?
+
+##### Ask the following questions to the group or student
+
+##### What is a microservices' architecture?
```
Comment (MSilva95) [inline subjects/devops/play-with-containers-py/audit/README.md:53]: extra apostrophe after "microservices."

File: subjects/devops/play-with-containers-py/audit/README.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+###### Did the student reply correctly to the questions?
+
+#### Verify the Dockerfiles:
+
+###### Is there a Dockerfile for each service?
+
+###### Are all Dockerfiles based on `Debian` or `Alpine`?
+
+###### There is no sensitive data in Dockerfiles or other solution files (sensitive data should only exist in `.env` file)?
```
Comment (MSilva95) [inline subjects/devops/play-with-containers-py/audit/README.md:69]: tis is a bit hard to read/understand, what do you think of this:
```Are Dockerfiles or any other solution files free from sensitive data (sensitive data should only exist in `.env` file)?```

File: subjects/devops/play-with-containers-py/audit/README.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+###### Is there a Docker image for each service with the same service name?
+
+##### Ask the following questions to the group or student
+
+##### What is a Docker image?
+
+##### Why do we use Docker images?
+
+##### Where can we find some public Docker images?
```
Comment (MSilva95) [inline subjects/devops/play-with-containers-py/audit/README.md:171]: this question is not explicit in the readme, the student wont know that this is a "must know"

File: subjects/devops/play-with-containers-py/audit/README.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
+#### General
+
+##### Check the Repo content
+
+A `README.md` file and all files used to create, delete and manage the student infrastructure must be submitted in the repo.
+
+###### Are all the required files present?
+
+###### Does the `.env` file not exist in the repo?
```
Comment (MSilva95) [inline subjects/devops/play-with-containers-py/audit/README.md:9]: maybe this is better:
```"Is the .env file absent within the repository?```

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Just the two sentences that in my opinion will cause some misunderstanding for students, and one of the questions is not mentioned in the readme so I'm not sure if it should exist in the audit

File: subjects/devops/play-with-containers-py/audit/README.md (hunk 3 above)
Comment (nprimo) [reply subjects/devops/play-with-containers-py/audit/README.md:171]: I see what you mean :) I will remove the question to be sure people won't think it is unfair

File: subjects/devops/play-with-containers-py/audit/README.md (hunk 4 above)
Comment (nprimo) [reply subjects/devops/play-with-containers-py/audit/README.md:9]: I opted for a slightly different version of yours :)

File: subjects/devops/play-with-containers-py/audit/README.md (hunk 1 above)
Comment (nprimo) [reply subjects/devops/play-with-containers-py/audit/README.md:53]: It is my grammar autocorrect that suggested adding the apostrophe :)

**Comment by nprimo** [comment]
I move this changes to [this PR](https://github.com/01-edu/public/pull/2277)

**Review by MSilva95 — COMMENTED** [review]
I will close this as it was moved, the changes are corrected in the other pr!

**Pull request closed without merging**